src/mopidy_mpd/protocol/music_db.py

```diff
@@ -532,7 +561,7 @@ def update(context, uri=None):
 
 # TODO: add at least reflection tests before adding NotImplemented version
 # @protocol.commands.add('readcomments')
-def readcomments(context, uri):
+def readcomments(context: MpdContext, uri: Uri | None = None) -> None:
```

Comment:
I agree that I don't really see what `readcomments` without an URI means. I think I typed it as optional because the docstring used `[URI]` instead of `{URI}`. I'll make it non-optional for now, and assume we'll revisit the official docs before ever implementing this.